### ISSUE TITLE
Bug- Adding a new meetup without filling time throws an error

### ISSUE THREAD

**Issue by poojithansl** [anitab-org/portal#286]
Time field must be required too. 
@tapasweni-pathak Currently there's only starting time, but no end time for an event/meetup.
How about adding a new field for end time to the model?

**Comment by aashutoshrathi** [comment]
Is this one fixed? Anyone working on it?

**Comment by abhi20161997** [comment]
@Tlazypanda please comment and seek permission on issues before starting to work on them

**Comment by satya7289** [comment]
Duplicate of the [issue](https://github.com/anitab-org/portal/issues/432).
